config/routes.rb

```diff
@@ -10,7 +10,7 @@
   namespace :api do
     namespace :v1 do
       resources :recipes, only: [:index, :show]
-      resources :recipe_builder, only: [:create]
+      resources :recipe_builders, only: [:create, :update]
```

Comment:
good update to have, the fact that you took on an update route as well is amazing, thank you for this!